_component_design/file-tile.md

```diff
@@ -0,0 +1,316 @@
+---
+title: File Tiles
+parent: Content Areas
+layout: component
+category: Components
+usage: >
+  A file tile displays the name, type, and size of an uploaded file. An uploaded file can be downloaded by a user.
+preview-image: preview-images/checkboxes.svg
```

Comment:
Preview image needs to be created for File Tiles.  There's a sketch file on GDrive which contains all the preview images.